content/project/conduct.adoc

```diff
@@ -7,180 +7,246 @@ section: conduct
+sustained inappropriate behavior.
 
-If probation clearly doesn't address the issue, or the issue is of high
-severity to warrant an expulsion, the contributor will be expelled from the
-Jenkins community for a period of 12 months. After which they may appeal to the
-board for the ban to be lifted.
+**Consequence**: A temporary ban from any sort of interaction or public
+communication with the community for a specified period of time
+(chats, mailing lists, pull requests, issues, events, etc.).
+No public or
+private interaction with the people involved, including unsolicited interaction
+with those enforcing the Code of Conduct, is allowed during this period.
+Violating these terms may lead to a ban.
 
-The ban will include but is not limited to:
+=== 4. Ban
 
-*  Bans from Jenkins community IRC Channels
-*  Deletion of their LDAP account
-*  Blocking their GitHub username from the jenkinsci github organization
-*  Banning their email address from jenkins mailing lists
+//oleg_nenashev: There is a deviation from Contributor Covenant 2.0 which recommends a Permanent Ban.
+// Jenkins Code of Conduct in the 2016 edition offered a way to restore accounts after 12 months in the case of explusion,
+// and it is retained in this edition
+
+**Community Impact**: Demonstrating a pattern of violation of community
+standards, including sustained inappropriate behavior,  harassment of an
+individual, or aggression toward or disparagement of classes of individuals.
 
+**Consequence**: A permanent ban from any sort of public interaction within
+the community.
+// Addition to Contributor Covenant 2.0 begins here
+The individual will be expelled from the Jenkins community.
+After 12 months they may appeal to the board for the ban to be lifted.
 
+The ban will include but is not limited to:
 
-NOTE: This page has been imported from the
-link:https://wiki.jenkins.io/display/JENKINS/Code+of+Conduct[Code of
-Conduct] wiki page, which, as of `v15`, was approved by the project governance
-meeting on
-link:http://meetings.jenkins-ci.org/jenkins-meeting/2016/jenkins-meeting.2016-01-06-19.01.html[2016-01-06]
+*  Bans from Jenkins community link:/chat[chats]
+*  Deletion of their LDAP account
+*  Blocking their GitHub username from the Jenkins GitHub organizations
+*  Banning their email address from Jenkins mailing lists
+*  Banning them from social media and meetup groups
+*  Banning them from participating in Community-organized events
+
+== Attribution
+
+This Code of Conduct is adapted from the Contributor Covenant,
+version 2.0, available at
+https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+
+Community Impact Guidelines were inspired by link:https://github.com/mozilla/diversity[Mozilla's code of conduct
+enforcement ladder].
+
+== Version history
+
+* **Jul TODO, 2020** - Major update of the Code of Conduct.
+  It was approved the project governance meeting, see the meeting notes link:TODO[here].
```

Comment:
Include the link to the governance meeting and a minor rephrasing of the sentence.
```suggestion
  It was approved in the project governance meeting, see the meeting notes link:https://docs.google.com/document/d/11Nr8QpqYgBiZjORplL_3Zkwys2qK1vEvK-NYyYa4rzg/edit#bookmark=id.yr5dfh4k94u6[here].
```

Reply:
Nope. It will be finally approved at the next meeting (hopefully)